Gemfile

```diff
@@ -1,9 +1,11 @@
 source "https://rubygems.org"
 git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
+gem "bcrypt", "3.1.12"
```

Comment:
gem này để làm gì đấy?

Reply:
gem này để băm password người dùng nhập, lưu lại giá trị băm trong csdl (sử dụng thuật toán bcrypt)